File: Data Structures & Algorithms/evaluate-reverse-polish-notation/submission-0.cpp
class Solution {
public:
    int evalRPN(vector<string>& tokens) {
        stack<int>st;
        int n=tokens.size();
        for(string &c:tokens)
        {
            if(c=="+"|| c=="-" || c=="*" || c=="/")
            {
                int b=st.top();
                st.pop();
                int a=st.top();
                st.pop();
                int ans;
                if(c == "+") ans = a + b;
                else if(c == "-") ans = a - b;
                else if(c == "*") ans = a * b;
                else ans = a / b; 
                st.push(ans);
            }
            else
            {
                st.push(stoi(c));
            }
        }
        return st.top();
        
    }
};

File: Data Structures & Algorithms/pow-x-n/submission-0.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        ans=1
        if n<0:
            n=-n
            x=1/x
        while n:
            if n%2!=0:
                ans*=x
            x*=x
            n//=2
        
        return ans
        